deploy/scripts/002_deploy_triggers.ts

```diff
@@ -0,0 +1,58 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { ethers, getChainId } from "hardhat";
+import { Contract } from "ethers";
+import { addToWhitelist } from "../utils";
+import dotenv from "dotenv";
+
+const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
+  dotenv.config({ path: (await getChainId()) == "31337" ? ".test.env" : ".env" });
+
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const whitelistService = await ethers.getContract("WhitelistService");
+  let trigWlHash = await whitelistService.getWhitelistHash(deployer, "triggers");
+  console.log("trigWlHash", trigWlHash);
```

Comment:
remove this

Reply:
Why? We need to know what the `trigWlHash` so we can subsequently add/remove from that whitelist using our multiSig no?
Granted, it shouldn't be a console.log and be stored somewhere more permanent. If that's the issue, I can leave this be and open an issue instead.

Reply:
hhmm why not store it in some variable and make it available to tests?

Reply:
Agreed that this isn't the way to do it. Just a dirty hack. Let me find a way.